features/support/paths.rb

```diff
@@ -82,7 +82,7 @@ def path_to(page_name, *args) # rubocop:disable Metrics/AbcSize, Metrics/Cycloma
       provider_admin_messages_outbox_index_path
 
     when /^the provider page of message with subject "([^"]*)"$/
-      message = @provider.sent_messages.find { |m| m.subject = $1 }
+      message = @provider.sent_messages.find { |m| m.subject == $1 }
```

Comment:
```suggestion
      message = @provider.sent_messages.find_by(subject: Regexp.last_match(1))
```

Reply:
The `$1` replacement is to make Rubocop happy.

Also, to fix the test, also the scenario needs to be fixed:

```
diff --git a/features/provider/admin/messages/outbox.feature b/features/provider/admin/messages/outbox.feature
--- a/features/provider/admin/messages/outbox.feature	(revision Staged)
+++ b/features/provider/admin/messages/outbox.feature	(date 1697192760609)
@@ -43,7 +43,7 @@
     Scenario: Reading a message
       Given they go to the provider sent messages page
       When follow "Welcome"
-      Then the current page is the provider page of message with subject "Welcome Alice"
+      Then the current page is the provider page of message with subject "Welcome"
 
     Scenario: Bulk operations
       Given they go to the provider sent messages page
```